static-analysis/autodispose-error-prone/src/main/java/com/uber/autodispose/errorprone/UseAutoDispose.java

```diff
@@ -92,8 +94,9 @@ public UseAutoDispose() {
   public UseAutoDispose(ErrorProneFlags flags) {
     Optional<ImmutableSet<String>> inputClasses =
         flags.getList("TypesWithScope").map(ImmutableSet::copyOf);
+    Optional<Boolean> exclusiveScope = flags.getBoolean("IsScopeExclusive");
```

Comment:
Open for other names. `IsScopeExclusive` is a misnomer coz there can be multiple scopes. But then `AreScopesExclusive` is a misnomer coz there can be only one custom scope 😄

Reply:
Let's go with `OverrideScopes`